src/ucp/wireup/wireup_ep.c

```diff
@@ -720,6 +720,20 @@ int ucp_wireup_ep_test(uct_ep_h uct_ep)
                     UCS_CLASS_DELETE_FUNC_NAME(ucp_wireup_ep_t);
 }
 
+static int ucp_wireup_aux_ep_is_owner(ucp_wireup_ep_t *wireup_ep,
+                                      uct_ep_h owned_ep)
+{
+    ucp_ep_h ucp_ep              = wireup_ep->super.ucp_ep;
+    ucp_lane_index_t cm_lane_idx = ucp_ep_get_cm_lane(ucp_ep);
+
+    return (wireup_ep->aux_ep == owned_ep) ||
+           /* Auxilliary EP can be WIREUP EP in case of it is on CM lane */
+           ((wireup_ep->aux_ep != NULL) &&
+            (cm_lane_idx != UCP_NULL_LANE) &&
+            (ucp_ep->uct_eps[cm_lane_idx] == &wireup_ep->super.super) &&
+            ucp_wireup_ep_is_owner(wireup_ep->aux_ep, owned_ep));
```

Comment:
what is the case where wireup_ep->aux_ep is also a wireup ep?

Reply:
I added the comment above
`Auxilliary EP can be WIREUP EP in case of it is on CM lane`